testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/arquillian/containers/KeycloakQuarkusServerDeployableContainer.java

```diff
@@ -150,7 +150,8 @@ private Process startContainer() throws IOException {
     private String[] getProcessCommands() {
         List<String> commands = new ArrayList<>();
 
-        commands.add("./kc.sh");
+        String os = System.getProperty("os.name").toLowerCase();
```

Comment:
Works for me to not add the SystemUtils library (or the Environment wrapper we use and you used in your previous version), but then please check if "os.name" is null before lowercasing, bc. even when true for 99% of the systems, it may be (and I experienced this in the past 😅 ) that os.name is not set at all for some reason.  Apache commons [checks](https://github.com/apache/commons-lang/blob/2b13f5b9fd288e4cce3300a6644181f67d6fc3a1/src/main/java/org/apache/commons/lang3/SystemUtils.java#L1908-L1913) for that, and so should we :)

Reply:
Thanks @DGuhr, I agree it can happen. I have added the suggested constraint.